core/commands/swarm.go

```diff
@@ -90,10 +90,13 @@ var swarmPeersCmd = &cmds.Command{
 				Peer: pid.Pretty(),
 			}
 
-			swcon, ok := c.(*swarm.Conn)
-			if ok {
-				ci.Muxer = fmt.Sprintf("%T", swcon.StreamConn().Conn())
-			}
+			/*
+				// FIXME(steb):
+							swcon, ok := c.(*swarm.Conn)
```

Comment:
definitely no need to reproduce the exact output here. But maybe we can print out the full transport.String() or something?

Reply:
Unfortunately, that won't include information about the muxer. Currently, we'd show:

```
<swarm.Conn[TCP] /local/maddr (<peer.ID local>) <-> /remote/maddr (<peer.ID remote>)>
```

(which should actually be cleaned up a bit...)

I've been toying with adding an `Info` function to transport conns (modeled after `Context.Value`):

```go
func Info(key interface{}) interface{}
```

This would allow querying for arbitrary information.